From c42dac05deef2bce6bd3d99871fc051982c1121b Mon Sep 17 00:00:00 2001
From: LuK1337 <priv.luk@gmail.com>
Date: Sun, 26 Aug 2018 15:00:21 +0200
Subject: [PATCH 32/33] Settings: Add LineageOS entries into device info

Author: Harry Youd <harry@harryyoud.co.uk>
Date:   Fri Nov 24 09:49:51 2017 -0800

    Settings: Add LineageOS version, API level and build date to device info

    Change-Id: I4acf834442712954f8e9db15370bffafd4547174

Author: Tobias Tefke <tobias.tefke@tutanota.com>
Date:   Wed Jun 13 11:27:22 2018 +0200

    [1/2] Add vendor security patch level to device info

    Change-Id: I523fcce65cc4155591c12ea1ac04807ebfc5c716

Change-Id: I8d4c6869f8dfa34a96567bec18f9f8276b66a64e
---
 res/layout/dialog_firmware_version.xml | 40 ++++++++++++++++++++++++++
 1 file changed, 40 insertions(+)

diff --git a/res/layout/dialog_firmware_version.xml b/res/layout/dialog_firmware_version.xml
index 874d7af006..1c27eb8c0e 100644
--- a/res/layout/dialog_firmware_version.xml
+++ b/res/layout/dialog_firmware_version.xml
@@ -37,6 +37,26 @@
             android:layout_width="match_parent"
             android:layout_height="wrap_content"/>
 
+        <TextView
+            style="@style/device_info_dialog_label"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:text="@*lineageos.platform:string/lineage_version"/>
+        <org.lineageos.internal.preference.deviceinfo.LineageVersionTextView
+            style="@style/device_info_dialog_value"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"/>
+
+        <TextView
+            style="@style/device_info_dialog_label"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:text="@*lineageos.platform:string/lineage_api_level"/>
+        <org.lineageos.internal.preference.deviceinfo.LineageAPIVersionTextView
+            style="@style/device_info_dialog_value"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"/>
+
         <TextView
             style="@style/device_info_dialog_label"
             android:id="@+id/security_patch_level_label"
@@ -51,6 +71,16 @@
             android:background="?android:attr/selectableItemBackground"
             android:textColor="?android:attr/colorAccent"/>
 
+        <TextView
+            style="@style/device_info_dialog_label"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:text="@*lineageos.platform:string/lineage_vendor_security_patch"/>
+        <org.lineageos.internal.preference.deviceinfo.LineageVendorSecurityPatchTextView
+            style="@style/device_info_dialog_value"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"/>
+
         <TextView
             style="@style/device_info_dialog_label"
             android:id="@+id/baseband_version_label"
@@ -75,6 +105,16 @@
             android:layout_width="match_parent"
             android:layout_height="wrap_content"/>
 
+        <TextView
+            style="@style/device_info_dialog_label"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:text="@*lineageos.platform:string/build_date"/>
+        <org.lineageos.internal.preference.deviceinfo.LineageBuildDateTextView
+            style="@style/device_info_dialog_value"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"/>
+
         <TextView
             style="@style/device_info_dialog_label"
             android:id="@+id/build_number_label"
-- 
2.17.1

